### ISSUE TITLE
Document how to upgrade Wazuh

### ISSUE THREAD

**Issue by jesuslinares** [wazuh/wazuh-kubernetes#3]

**Comment by SitoRBJ** [comment]
Hello team,

We have tried to perform an update on Kubernetes of the Wazuh cluster. 

We have made a first basic installation, as it comes in our guide, using for the Wazuh containers the image: 

- wazuh/wazuh:3.6.1_6.4.0

Later, we have added for testing a set of rules in our `local_rules.xml` file located in `/var/ossec/etc/rules` path from inside the pod.  This action has modified the `local_rules.xml` file  in the `/var/ossec/data/etc/rules` path and in the `/etc/postfix/etc/rules` path. 

This happens because they are the volumes we assemble. For example, in the wazuh manager master:

```
          volumeMounts:
            - name: config
              mountPath: /wazuh-config-mount/etc/ossec.conf
              subPath: ossec.conf
              readOnly: true
            - name: wazuh-manager-master
              mountPath: /var/ossec/data
            - name: wazuh-manager-master
              mountPath: /etc/postfix
```

As we use Kubernetes with Docker service, the files and directories exported to the volumes are those we use in our Docker images, specifically those used for the version of the installed image are: 

```
DATA_DIRS[((i++))]="etc"
DATA_DIRS[((i++))]="ruleset"
DATA_DIRS[((i++))]="logs"
DATA_DIRS[((i++))]="stats"
DATA_DIRS[((i++))]="queue"
DATA_DIRS[((i++))]="var/db"
DATA_DIRS[((i++))]="api"
```

Now we proceed to the update. As we are aware of certain errors in Docker when updating due to certain files that were exported in previous versions, we had to adjust the content of the volumes. 

As we are going to install version:
- wazuh/wazuh 3.7.0_6.5.0

we adapt the content of the volumes in `/var/ossec/data`:

```
DATA_DIRS[((i++))]="api/configuration"
DATA_DIRS[((i++))]="etc"
DATA_DIRS[((i++))]="logs"
DATA_DIRS[((i++))]="queue/db"
DATA_DIRS[((i++))]="queue/rootcheck"
DATA_DIRS[((i++))]="queue/agent-groups"
DATA_DIRS[((i++))]="queue/agent-info"
DATA_DIRS[((i++))]="queue/agents-timestamp"
DATA_DIRS[((i++))]="queue/agentless"
DATA_DIRS[((i++))]="queue/cluster"
DATA_DIRS[((i++))]="queue/rids"
DATA_DIRS[((i++))]="queue/fts"
DATA_DIRS[((i++))]="var/multigroups"
```

We have removed unnecessary directories and files and updated the `/etc/internal_options.conf` file.

Note: These steps will not be necessary in the future, when Docker performs the updates correctly. 

Once the volumes are ready we proceed to update the 3 pods of the Wazuh managers.

It is enough to perform these two steps with each pod. 

1. Change the image of the container in `wazuh-manager-master-sts.yaml`, `wazuh-manager-worker-0-sts.yaml`, and `wazuh-manager-worker-0-sts.yaml`  files.

```
      containers:
        - name: wazuh-manager
          image: 'wazuh/wazuh:3.7.0_6.5.0'
```

2. Apply the new configuration. 

```
ubuntu@k8s-control-server:~/wazuh-kubernetes/manager_cluster$ kubectl apply -f wazuh-manager-master-sts.yaml
statefulset.apps "wazuh-manager-master" configured
```

This process will end the old pod while creating a new one, linked to the same volume. 

Once the new pods are running, we can check that the Wazuh managers are in version 3.7.0, that the cluster is still working and that the rules we have created in test mode are still there. 

Regards,

Alfonso Ruiz-Bravo

**Comment by SitoRBJ** [comment]
Hello team

This issue is solved in PR #6 

Regards,

Alfonso Ruiz-Bravo
